thrall/app/lib/ElasticSearch.scala

```diff
@@ -165,10 +171,16 @@ object ElasticSearch extends ElasticSearchClient with ImageFields {
         scriptType)
       .executeAndLog(s"setting collections on image $id")
       .incrementOnFailure(failedCollectionsUpdates) { case e: VersionConflictEngineException => true }
+    )
+
+  def prepareImageUpdate(id: String): List[UpdateRequestBuilder] =
+    getCurrentIndices.map(_.map (index =>
+      client.prepareUpdate(index, imageType, id)
+        .setScriptLang("groovy")
+      )
+    ).get
```

Comment:
Having a get is going to throw on an empty Option. It might be worth thinking about where we throw, potentially in `getCurrentIndices` because that should always return something?